sql/src/test/java/io/crate/integrationtests/RestSQLActionIntegrationTest.java

```diff
@@ -80,4 +80,11 @@ public void testSetCustomSchema() throws IOException {
         assertThat(response.getStatusLine().getStatusCode(), is(404));
         assertThat(EntityUtils.toString(response.getEntity()), containsString("TableUnknownException"));
     }
+
+    @Test
+    public void testExecutionErrorContainsStackTrace() throws Exception {
+        CloseableHttpResponse resp = post("{\"stmt\": \"select 1 / (height - 4808) from sys.summits\"}");
```

Comment:
You're right, changed it. For some reason I assumed it had to be a runtime error because in `protocols/http.txt` we have an example how the error_trace can be enabled and that example worked.